src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js

```diff
@@ -50,6 +50,11 @@ class CJ4_FMC extends FMCMainDisplay {
         this.selectedWaypoint = undefined;
         this.selectMode = CJ4_FMC_LegsPage.SELECT_MODE.NONE;
         SimVar.SetSimVarValue("TRANSPONDER STATE:1", "Enum", 1);
+        this.cj4Units = WTDataStore.get('WT_CJ4_Units', 0);
+        SimVar.SetSimVarValue("L:WT_CJ4_Units", "Enum", this.cj4Units); //0 = imperial; 1 = metric
+        this.cj4Weight = this.cj4Units == 1 ? 0.453592 : 1; //default sim value for weight is lbs
```

Comment:
is all this still needed? line 53-57

Reply:
good catch! need to delete all of those! don't need the simvar anymore.